lib/ansible/utils/module_docs_fragments/gcp.py

```diff
@@ -0,0 +1,45 @@
+# Copyright: (c) 2018, Google Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+        # GCP doc fragment.
+        DOCUMENTATION = '''
+options:
```

Comment:
Rather than defining these options in every modules argspec the should be defined once in `module_utils/gcp_utils.py`

http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_in_groups.html#your-first-pull-request

Reply:
Is this necessary? We're in a strange place with Magic Modules (the code generator).

Magic Modules acts as the central repository for this information and has all of these options in one central file.

Reply:
I don't agree with moving shared documentation out of a fragment - the use of doc fragments also serves to keep docs consistent across modules, and in large part we treat generated modules the same way we treat regular ones. This means that the same attention should be paid to code/doc sharing via mod_utils and doc fragments and so forth.

Reply:
Yup, please split this into docs fragment

Reply:
Sounds good. The documentation is currently in a document fragment. I'll consolidate the shared module argspec in module_utils/gcp_utils.py

Reply:
Change is made. Documentation uses fragments and module argspec is now centralized in module_utils/gcp_utils.py